### ISSUE TITLE
Deceleration/Friction too slow, and accelerated back hopping can be performed without ever looking forward first

### ISSUE THREAD

**Issue by googleooer** [ProjectBorealis/PBCharacterMovement#36]

**Comment by mastercoms** [comment]
None of these are different from the base game. We tested 1 to 1 with HL2Ep2 and also consulted a dozen experienced speedrunners, some of which have participated in GDQ.

You don't have to look forward before ABHing. There's no stateful system which would even be able to track this.

Any perceived issues are due to Unreal's slightly different key/mouse sampling system and unfixed game tick rate, which make accomplishing some tasks different.

Friction is practically the same but may have slight substepping issues due to framerate. But we measured this to be within a very small error, so it's not something we're investigating at this time.
